src/snackbar/snackbar.ios.ts

```diff
@@ -113,8 +137,11 @@ export class SnackBar extends SnackBarBase {
         if (options.hideDelay > 10) {
             options.hideDelay = 10;
         }
-        const snackbarManager = (this._snackbarManager = MDCSnackbarManager.defaultManager);
-        snackbarManager.delegate = MDCSnackbarManagerDelegateImpl.initWithOwner(new WeakRef(this));
+        const snackbarManager = (this._snackbarManager =
+            MDCSnackbarManager.defaultManager);
+        snackbarManager.delegate = MDCSnackbarManagerDelegateImpl.initWithOwner(
```

Comment:
I think would be better when the settings are in 120, but regarding those lines, it seems redundant having a const with the same value when you have `this. _snackbarManager `
this may have problems with the delegate, one way to solve is to save the delegate in a class member variable like `this._snackbarManagerDelegate ` and then set it to null ones the view is destroyed

Reply:
@henrychavez agree. I am far from finished on the snackbar implementation :D 
Again would prefer this to be in another work load if you don't mind